dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteFilesPopup.java

```diff
@@ -37,22 +36,13 @@
 
     public static void show(final Ds3Task deleteTask, final Ds3Common ds3Common) {
         final DeleteItemView deleteView = new DeleteItemView(deleteTask);
-        final TreeTableView<Ds3TreeTableValue> ds3TreeTableView = ds3Common.getDs3TreeTableView();
-        if (ds3TreeTableView != null) {
-
+        if (ds3Common.getDs3TreeTableView() != null) {
             ObservableList<TreeItem<Ds3TreeTableValue>> selectedPanelItems = ds3Common.getDs3TreeTableView().getSelectionModel().getSelectedItems();
```

Comment:
I know that this is not your code, but is this missing a `final`

Reply:
It is reassigned  as an empty FXCollections.observableArrayList 2 lines down, if it is... a null or empty Observable list.  Not sure what they're trying to prevent there, don't think we can launch the DeletePopup without a selection